Documentation/DE/Referenz/Hooks/Index.rst

```diff
@@ -16,115 +16,330 @@
+        {
+            // Hier Ihr Code
+        }
+    }
 
-\*  **@return** void
+.. _listview_de:
 
-\*/
+Hooks zur Listenansicht
+"""""""""""""""""""""""
 
-public functionmodifyEventSingleView(tx\_seminars\_Model\_Event$event,
-Tx\_Oelib\_Template$template) {…}
+.. important::
+    Die Nutzung von :php:`\Tx_Seminars_Interface_Hook_EventListView` ist veraltet seit
+    seminars 3. Es wird in seminars 4 entfernt werden. Bitte aktualisieren Sie auf
+    :php:`\OliverKlee\Seminars\Hooks\Interfaces\SeminarListView`.
 
-/\*\*
+Es gibt 5 Hooks in die Listenansicht(en). Der erste Hook wird vor der Erstellung der
+Seminar-Bag (die in der Liste auszugebenden Seminare) oder der Registrierungen-Bag (die
+Seminare, für die sich ein Benutzer angemeldet hat) aufgerufen. Der Hook wird immer aufgerufen,
+auch wenn die Liste leer sein wird.
 
-\* Modifies a list row in the time slots list (which is part of the
-event
+Die übrigen Hooks werden während der Erstellung der Seminar-Listen-Tabelle aufgerufen:
 
-\* single view).
+* Bevor der Tabellenkopf in HTML umgewandelt wird
+* Bevor eine Tabellenzeile zu einem bestimmten Seminar oder einer Registrierung in HTML umgewandelt wird
+* Im Fall der `my_event` Liste: direkt nach dem oben genannten Zeilen-Hook
+* Bevor der Tabellenfuß in HTML umgewandelt wird
 
-\*
+In diesen Hooks können Sie eigene Marker ausfüllen oder vorhandene Marker-Werte ändern. Zu
+verfügbaren Eigenschaften und Methoden siehe :file:`Classes/Frontend/DefaultController.php`.
 
-\*  **@param** tx\_seminars\_Model\_TimeSlot $timeSlot
+Der Hook in die Erstellung der Seminar- oder Registrierungen-Bag erlaubt es, die für die Liste
+ausgewählten Seminare bzw. Reqistrierungen zu beeinflussen. Sie können neue Filter hinzufügen oder
+bestehende Filter entfernen. Details dazu finden Sie in :file:`Classes/BagBuilder/AbstractBagBuilder.php`,
+:file:`Classes/BagBuilder/Event.php` und :file:`Classes/BagBuilder/Registration.php`.
 
-\* the time slot to display in the current row
+Sie müssen 7 Listenarten bei Ihrer Implementation beachten:
 
-\*  **@param** Tx\_Oelib\_Template $template
+* Themen Liste (`topic_list`)
+* Termin Liste (`seminar_list`)
+* Meine Seminare (`my_events`)
+* Meine VIP Seminare (`my_vip_events`)
+* Von mir angelegte Seminare (`my_entered_events`)
+* Termine am nächsten Tag (`events_next_day`)
+* Andere Termine (`other_dates`)
 
-\* the template that will be used to create the list row output
+Die letzten beiden Listenarten (Termine am nächsten Tag und Andere Termine) gehören zur Einzelansicht,
+werden aber als komplette Listenansicht behandelt (inklusive Erstellung der Seminar-Bag).
 
-\*
+Ihre Klasse, die :php:`\OliverKlee\Seminars\Hooks\Interfaces\SeminarListView` implementiert,
+machen Sie seminars in :file:`ext_localconf.php` Ihrer Extension bekannt:
 
-\*  **@return** void
+.. code-block:: php
 
-\*/
+    use \OliverKlee\Seminars\Hooks\Interfaces\SeminarListView;
```

Comment:
For the hook registration (which happens in ext_localconf.php etc.), we cannot use classes AFAIK, and need to use FQCN.